src/app/shell/admin-tools/data/admins/admins.component.ts

```diff
@@ -204,11 +210,13 @@ export class AdminsComponent implements OnInit, OnDestroy {
 
   onPageChange(page: PaginationElement): void {
     this.currentPage = page;
-    this.store.dispatch([new OnPageChangeAdminTable(page), new GetAllMinistryAdmins()]);
+    Util.setPaginationParams(this.adminParams, page, this.adminParams.size);
+    this.store.dispatch([new OnPageChangeAdminTable(page), new GetAllMinistryAdmins(this.adminParams)]);
```

Comment:
this logic is duplicated, so create private method and move there duplicated logic